source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst

```diff
@@ -50,6 +50,7 @@ What's new
   - Important Bug Fixes on top of Processor SDK 10.00.07.04 Release
   - RT Kernel : Real-Time Linux Interrupt Latency numbers here - :ref:`RT Interrupt Latencies <RT-linux-performance>`
   - Power Management: DeepSleep, MCU-Only and Partial IO mode - :ref:`Power Management Overview <Power-Management>`
+  - Support for OV5640 + `Arducam V3Link (Fusion Mini) <https://www.arducam.com/product/arducam-v3link-camera-kit-for-ti-development-boards/>`_
```

Comment:
same here

Reply:
@devarsht I have made the changes, please review.